revision.h.in

```diff
@@ -18,11 +18,8 @@
 #else
 #define VER_FILEVERSION_STR         STR(VER_MAJOR) "." STR(VER_MINOR) "." STR(VER_PATCH) "." STR(VER_BUILD) " (Commit @GIT_COMMIT_HASH@)"
 #endif
+#define VER_COPYRIGHT_STR           "� 2006-@THIS_YEAR@"
```

Comment:
Avoid non-ascii characters in source code and especially string literals.
Not all editors support them - including github - and of course the native unicode string encoding varies between platforms.
Just use (C).

Reply:
Found ANSI escape sequence. Fixed.